core/bin/dust_api.rs

```diff
@@ -558,6 +558,7 @@ struct RunsCreatePayload {
     inputs: Option<Vec<Value>>,
     config: run::RunConfig,
     credentials: run::Credentials,
+    secrets: HashMap<String, String>,
```

Comment:
This should be an Option so that we don't break the current front<>core intergration

Reply:
Still applicable